code/src/components/Buttons.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { AiOutlineLeft } from "react-icons/ai";
+
+
+
+const Buttons = ({ nextQuestion, prevQuestion, questionNum}) => {
+    return <>
+    {questionNum === 1 && (
+    <button  className='btn next-btn'  type="button" onClick={nextQuestion}>Next</button>
+
+    )} 
+    {questionNum > 1 && questionNum < 6 && (
+    <>
+    <button className='btn prev-btn' type="button" onClick={prevQuestion}><AiOutlineLeft className='prev-icon'/></button>
+    <button className='btn next-btn' type="button" onClick={nextQuestion}>Next</button>
+    </>
+    )}
+
+    {questionNum === 6 && (
+    <>
+    <button className='btn prev-btn' type="button" onClick={prevQuestion}><AiOutlineLeft className='prev-icon'/></button>
+    </>
+    )}
+    </>
+}
+
+export default Buttons;
```

Comment:
Smart approach! I like it! The only thing I am thinking of is that it might be a bit cumbersome to remember to update in all components as well.